src/Symfony/Component/Translation/Tests/Catalogue/TargetOperationTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Tests\Catalogue;
+
+use Symfony\Component\Translation\Catalogue\TargetOperation;
+use Symfony\Component\Translation\MessageCatalogueInterface;
+
+class TargetOperationTest extends DiffOperationTest
```

Comment:
Inheritance should be the other way around. Otherwise, the new `TargetOperationTest` is a legacy test too.